src/test/kotlin/com/ing/dlt/zkkrypto/ecc/schnorr/SchnorrTest.kt

```diff
@@ -0,0 +1,180 @@
+package com.ing.dlt.zkkrypto.ecc.schnorr
+
+import com.ing.dlt.zkkrypto.ecc.EllipticCurvePoint
+import com.ing.dlt.zkkrypto.ecc.curves.AltBabyJubjub
+import com.ing.dlt.zkkrypto.ecc.curves.Jubjub
+import com.ing.dlt.zkkrypto.ecc.pedersenhash.PedersenHash
+import com.ing.dlt.zkkrypto.util.BitArray
+import org.junit.jupiter.api.Test
+import java.math.BigInteger
+
+
+class SchnorrSignatureTest {
+
+    @Test
+    fun `test raw message signing`(){
+
+        val schnorr = SchnorrSignature.zinc()
+
+        schnorr.generatePrivateKey()
+        schnorr.getPublicKeyFromPrivate()
+
+        println("Public Key:")
+        println("\tx coord = " + schnorr.publicKey.x.toString(16))
+        println("\ty coord = " + schnorr.publicKey.y.toString(16) + "\n")
+
+        val msg = 85.toByte()
+        val msgBytes = byteArrayOf(msg)
+
+        //val msg = "Foo bar pad to16"
```

Comment:
remove commented code or make two assertions in the test, one for 85 and one more ByteArray, for example
```
listOf(ByteArray(1){ 85 }, "Foo bar pad to16".toByteArray).forEach {
  ... verification logic
}
```